### ISSUE TITLE
Usability issues to tackle during the components fix

### ISSUE THREAD

**Issue by Shiny-Penny** [ONSvisual/local-statistics#160]
A list of the issues we've marked with "Should be fixed with new components"

- [ ] DAC_Illogical_Heading_Structure_Usability_01: 
Modify the "Find an area" and "Local indicators" headings on the opening page to be H2 rather than H3, that'll fix the structure. Quick win, and we can modify the CSS if need be. (log row: 6; report pg: 95)

- [ ] DAC_Incorrect_Use_Of_Aria_Usability_01:
Content including headings, input fields and buttons have been incorrectly placed inside a `<figure>` element. This may be confusing for screen reader users who hear ‘figure’ when they enter the section ahead of a h3 heading. The figure element should be used to specify self-contained content, like illustrations, diagrams, photos, code listings, etc.
<br>**DAC solution:** Remove the `<figure>` element as it is not appropriate for the content contained inside it and may be confusing for screen reader users. (log row: 9; report pg: 97)
